spec/models/invitation_spec.rb

```diff
@@ -78,5 +78,21 @@
         end
       end
     end
+
+    # Regression test for AO3-6094
```

Comment:
You can leave out this comment. We reference Jira issues in code comments for TODOs or tricky stuff that would come back to bite us, but generally not for regression tests.